draft-ietf-mls-protocol.md

```diff
@@ -1445,6 +1525,34 @@ the Group.
 It is RECOMMENDED for the application generating exported values
 to refresh those values after a group operation is processed.
 
+## Recovery Keys
+
+The main MLS key schedule provides a `recovery_secret` which can provide extra
+security in some cross-group operations.
+
+The application SHOULD specify an upper limit on the number of past
+epochs for which the `recovery_secret` may be stored.
+
+There are three ways in which a `recovery_secret` can be used to re-initialize
+the group with different parameters, to re-add lost group members or to create a
+sub-group of an existing group, as detailed in {{pre-shared-keys}}.
+
+Recovery keys are distinguished from exporter keys in that they have specific
+use inside the MLS protocol, whereas the use of exporter secrets may be
+decided by an external application. They are thus derived separately to avoid
+key material reuse resulting from poor exporter key management external to
+the protocol.
+
+## State Authentication Keys
+
+The main MLS key schedule provides a per-epoch `authentication_secret`
+which MAY be used for authenticating the current group state. As with recovery
+keys, these are distinguished from exporter keys in that they have specific
+use affecting the MLS protocol.
+
+Authentication keys MAY be used to authenticate current or past group
+members out-of-band.
```

Comment:
I think you definitely need to elaborate on what this means / how it would be done?

Reply:
I've added another sentence or two on what this is supposed to achieve and how.